style.css

```
@@ -0,0 +1,116 @@
+*{
+    box-sizing: border-box;
+    font-family: 'Poppins', sans-serif;
+    margin: 0;
+}
+
+.mainComponentContainer{
```

Add space between selector and bracket. Do this for all selectors in file.

```suggestion
.mainComponentContainer {
```